[CI] Merging PRs is blocked due to python SDK lint issue

### What happened?

PRs aren't being merged to master via the merge queue due to stap `Lint Python` failing even though the PR does unrelated changed to the python SDK or python tooling. The error:
```
. venv/*/activate && \
	python -m black ./lib/pulumi --exclude lib/pulumi/runtime/proto --check
All done! ✨ 🍰 ✨
51 files would be left unchanged.
. venv/*/activate && \
	MYPYPATH=./stubs python -m mypy ./lib/pulumi --config-file=mypy.ini
lib/pulumi/runtime/rpc.py:129: error: Module "google.protobuf.pyext" has no attribute "cpp_message"  [attr-defined]
Found 1 error in 1 file (checked 81 source files)
make: *** [Makefile:48: lint] Error 1
Error: Process completed with exit code 2.
```

### Example

N/A

### Output of `pulumi about`

N/A

### Additional context

_No response_

### Contributing

_No response_

--- a/sdk/python/lib/pulumi/runtime/rpc.py
+++ b/sdk/python/lib/pulumi/runtime/rpc.py
@@ -126,7 +126,7 @@ except ImportError:
 
 # New versions of protobuf have moved the above import to api_implementation
 try:
-    from google.protobuf.pyext import (
+    from google.protobuf.pyext import (  # type: ignore
         cpp_message,
     )  # pylint: disable-msg=E0611
 

